src/client/gitlab.ts

```diff
@@ -285,12 +288,17 @@ export const getProjectBranch = async (
   return branch;
 };
 
-export const getOwnedProjectsBySearchCriteria = async (
+/**
+ * Get project data that the token role is at least a maintainer of, filtered by search criteria.
+ * @param search - search criteria i.e. project name
+ * @param groupToken - Gitlab access token
+ */
+export const getMaintainedProjectsBySearchCriteria = async (
   search: string,
   groupToken: string,
 ): Promise<GitlabAPIProject[]> => {
   const params = {
-    owned: 'true',
```

Comment:
Is it possible to keep the existing orchestration if the feature flag is off, and still filter by owner in this flow?

Do we have a feeling of how many additional matches, if any, the new search query would fetch in an average query compared to the existing orchestration?

Reply:
I actually don't see any reference to a feature flag, unless that was introduced a previous PR. Is the thought to release support for Maintainer tokens and deprecate the functionality around Owner or are we allowing both types going forward?

Reply:
I think the intention is to allow both, but gate it behind a FF for now

Reply:
@eserranovalle The query still uses the search string which reasonably limits the search space - likely only to duplicates with same/similar project name. Based on walking through this code execution with logging, I'd say this is a safe change.

And yes, the intention is to allow both. The newer flow will be gated via a UI feature flag.

Reply:
Following further discussion, this segment is gated behind FF.